client/settings/fraud-protection/advanced-settings/rule-card-notice.tsx

```diff
@@ -28,7 +35,7 @@ const FraudProtectionRuleCardNotice = ( { type, children } ) => {
 
 	return (
 		<BannerNotice
-			status={ type }
+			status={ type } // Type 'string' is not assignable to type 'Status | undefined'.
```

Comment:
Is this a todo comment?

Reply:
No I think it was just a comment we put there as we were dealing with some TS linting issues with it and a related test at the time. I guess we just forgot to remove it. Thanks for flagging. Removed in [3fba24d](https://github.com/Automattic/woocommerce-payments/pull/6032/commits/3fba24df9ea4c5e43daf0ee2ffa7695457f13f15).